todo_app/user/user.py

```diff
@@ -0,0 +1,30 @@
+from flask_login import UserMixin
+from enum import Enum
+from functools import wraps
+from flask_login import current_user
+
+
+class User(UserMixin):
+    def __init__(self, id):
+        self.id = id
+        self.role = user_mapping[id] if id in user_mapping.keys() else None
```

Comment:
It's feel a little odd that you fall back to None rather than Reader, and that the Reader element of the Enum is never used, although I agree it functionally makes no difference